src/client/app/i18n.ts

```diff
@@ -2,12 +2,23 @@ import i18next from 'i18next'
 import Backend from 'i18next-http-backend'
 import { initReactI18next } from 'react-i18next'
 
+declare let ASSET_VARIANT: string
+const assetVariant = ASSET_VARIANT || 'gov'
```

Comment:
Surely...
```suggestion
const assetVariant = process.env.ASSET_VARIANT || 'gov'
```

And then key the value in DefinePlugin as `process.env.ASSET_VARIANT`?